docker-compose.yml

```diff
@@ -10,9 +10,16 @@ services:
     container_name: mysql-db
     env_file:
       - ./packages/server/.env
+        
+    volumes:
+      - mysql-data:/var/lib/mysql
```

Comment:
Where this folder will be created? If inside the project then we should put this in .gitignore.

Reply:
No it will be created in the device

Reply:
Okay